medusa/name_parser/parser.py

```diff
@@ -279,6 +279,12 @@ def _parse_series(result):
         new_absolute_numbers = []
 
         ex_season = scene_exceptions.get_season_from_name(result.series, result.series_name) or result.season_number
+        if ex_season == None:
+            ex_season = 1
+            log.info(
```

Comment:
Won't this break support for most specials?

Reply:
Didn't think of that. I'll check it out